### DISCUSSION TITLE
Changing the vid_stride parameter

### DISCUSSION THREAD

**Discussion by ImsaraSamarasinghe** [ultralytics/ultralytics#4672] [Q&A]
Hey people,
I'm pretty new to the Ultralytics library and have been playing around with the yoloV8 Native Tracking. The tracking for my use case works quite well, however, I think skipping a few frames from my video source might help with performance. I tried to change the parameter vid_stride in ultralytics\data\loaders.py , but it didn't help at all. Is it possible to change the vid_stride this way? or is there another method involved?
thanks

**Comment by glenn-jocher** [comment]
@ImsaraSamarasinghe hello there! 👋

Great to hear you're exploring YOLOv8 Native Tracking! To adjust the frame skipping (vid_stride), you should indeed modify the `vid_stride` parameter. However, ensure that you're changing it in the correct location and in the configuration that's being used at runtime.

If directly modifying `loaders.py` didn't work, it's possible that your changes weren't picked up due to caching or you might be editing the wrong instance of the file. Instead of changing the source code, you might want to pass the `vid_stride` parameter through your command line call or within your custom configuration file, if that's how you're running the model.

Remember to restart your environment to ensure all changes are applied. If you continue to face issues, double-check the documentation for the latest guidance on configuring tracking parameters.

Happy tracking! 😊
